CHANGELOG.rst

```diff
@@ -11,6 +11,12 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[1.2.1] - 2020-12-17
+~~~~~~~~~~~~~~~~~~~~
+
+* Improve monitoring of legacy Waffle class imports. We should watch for "edx_toggles.toggles.waffle.legacy.internal.WaffleSwitch" custom attributes.
```

Comment:
If you want it to be accurate, it should be: "edx_toggles.toggles.internal.waffle.legacy"